client/src/pages/EpisodeDetailsPage.js

```diff
@@ -116,6 +119,8 @@ function EpisodeDetailsPage() {
   const { id } = useParams();
   const { user } = useAuth();
   const [episodeDetails, setEpisodeDetails] = useState(null);
+  const [isLiked, setIsLiked] = useState(null);
```

Comment:
Would expect a boolean, as ``isLiked`` starts with ``is``, but initial value is null. Maybe it is applicable to set the value to ``false``?